vignettes/getting-started.Rmd

```diff
@@ -17,13 +17,65 @@ knitr::opts_chunk$set(
 # gimap tutorial
 
 ```{r}
+library(magrittr)
 library(gimap)
 ```
 
 ```{r}
-data <- example_data()
+example_data <- example_data()
+
+# Let's examine this example metadata
+example_data
+```
+
+```{r}
+colnames(example_data)
 ```
 
+## Setting up data
+
+We're going to set up three datasets. The first is required, its the counts that the genetic interaction analysis will be used for. 
+
+```{r}
+example_counts <- example_data %>%
+  dplyr::select(c("Day00_RepA", "Day05_RepA", "Day22_RepA", "Day22_RepB", "Day22_RepC")) %>%
+  as.matrix()
+```
+
+The next two datasets are metadata that describe the dimensions of the count data. 
+- These both need to be data frames. 
+- The sizes of these metadata must correspond to the dimensions of the counts data. 
+- The first column of the pg_metadata must be a unique id
```

Comment:
Do we need to set up a check for this in the `setup_data()` function?

Reply:
Yes! Good call. I'll add that.

Reply:
Added in https://github.com/FredHutch/gimap/pull/12/commits/655a7fb2ace7775ede0b91167c8a1bedfb2fb54d